official/wide_deep/wide_deep.py

```diff
@@ -218,5 +203,16 @@ def main(unused_argv):
 
 if __name__ == '__main__':
   tf.logging.set_verbosity(tf.logging.INFO)
+
+  parser = parsers.BaseParser()
+  parser.set_defaults(data_dir='/tmp/census_data',
+                      model_dir='/tmp/census_model',
+                      train_epochs=40,
+                      epochs_per_eval=2,
+                      batch_size=40)
+  parser.add_argument(
+      '--model_type', type=str, default='wide_deep',
+      help="Valid model types: {'wide', 'deep', 'wide_deep'}.")
```

Comment:
Prepend "[default %(default)s] " to the help string.

Reply:
Done